lib/pytap13.py

```diff
@@ -17,43 +17,37 @@
 # Author: Josef Skladanka <jskladan@redhat.com>
 
 import re
-
-try:
-    # Python 2
-    from StringIO import StringIO
-except ImportError:
-    # Python 3
-    from io import StringIO
+from io import BytesIO
 
 import yaml
-from lib.utils import string_types
 
+from lib.utils import string_types, binary_regex, str_to_bytes, bytes_to_str
```

Comment:
Use a separate line for each import (I don't remember where these rules come from, but we follow it).

Reply:
Ok. 
Thank you